lab-anthony/entry.js

```diff
@@ -0,0 +1,32 @@
+'use strict';
```

Comment:
In general, I would never write any actual code in a file called 'entry.' Entry files are backend code that webpack runs and should really only include require statements. All the stuff that you're doing to build your actual model should be broken out into a separate, appropriately named file and then required in.